src/graph/validator/Validator.h

```diff
@@ -203,6 +204,7 @@ class Validator {
   std::set<std::string> userDefinedVarNameList_;
   // vid's Type
   Value::Type vidType_;
+  static thread_local uint32_t maxStatements_;
```

Comment:
Why it need to be thread_local? Validator is single-thread.

Reply:
makeValidator is a static function，maxStatement_ is also static member, so use thread_local